samples/CMakeLists.txt

```diff
@@ -47,6 +47,15 @@ if (CONFIG_SECURE_BOOT)
       SOURCE_DIR ${CMAKE_CURRENT_LIST_DIR}/bootloader
       )
   endif()
+  if (CONFIG_SOC_NRF5340_CPUAPP AND CONFIG_BOOTLOADER_MCUBOOT)
```

Comment:
Can you check the actual PM size of MCUboot instead? Does that make sense?

Reply:
I don't think I understand what you mean. To check if the user has already changed the default?

Reply:
Hmm, what is it that makes `CONFIG_SOC_NRF5340_CPUAPP` too large, and not e.g. `CONFIG_SOC_NRF9160`?

Reply:
I haven't checked the diff, but I assume some soc startup stuff which occupies a bit more flash.

It only fails for nrf53.